rowboat/plugins/admin.py

```diff
@@ -471,21 +471,21 @@ def tempmute(self, event, user, duration, reason=None):
             if len({event.config.temp_mute_role, event.config.mute_role} & set(member.roles)):
                 raise CommandFail('{} is already muted'.format(member.user))
 
-            expire_dt = parse_duration(duration)
+            expires_dt = parse_duration(duration)
```

Comment:
Just generate the humanized version alongside here & store in a variable instead of regenerating twice below.